src/elements/common/sub-header/SubHeaderRight.tsx

```diff
@@ -91,9 +100,33 @@ const SubHeaderRight = ({
             )}
 
             {isMetadataView && isMetadataViewV2Feature && (
-                <Button icon={Pencil} size="large" variant="primary">
-                    {formatMessage(messages.metadata)}
-                </Button>
+                <>
+                    {(selectedItemIds === 'all' || (selectedItemIds instanceof Set && selectedItemIds.size > 0)) && (
+                        <DropdownMenu.Root>
+                            <DropdownMenu.Trigger className="be-sub-header-right-dropdown-trigger">
```

Comment:
new class names should follow suit naming. maybe something like:
`be-SubHeaderRight-actions` / `be-SubHeaderRight-actionsTrigger`